tests/test_cli.py

```diff
@@ -393,13 +395,15 @@ def my_resource(rtype, *args):
         result = runner.invoke(cli, ['print', 'myapp.yaml', '--region=myregion', '123', 'master-mind'],
                                catch_exceptions=False)
 
-    stdout, cfjson = result.output.split('\n', 1)
-    assert 'Generating Cloud Formation template.. OK' == stdout
-    awsjson = json.loads(cfjson)
+    output = io.StringIO(result.output)
```

Comment:
Just a personal preference so I don't need to duplicate the output contents in memory. Given that performance is not critical here both options would be ok.